src/xdplwf/recv.c

```diff
@@ -1602,6 +1637,11 @@ XdpGenericReceivePostInspectNbs(
             }
         }
 
+        //
+        // In low resource scenarios, only one NB should be processed at a time.
+        //
+		ASSERT(CanPend || NbHead == NbTail);
```

Comment:
I think there are some hard tabs hiding in here. You might want to adjust your editor to always use spaces.

Reply:
Ugh. I thought I solved that one, I don't know why on this specific project something sneaks tabs in. Hopefully I have my settings right now :/